src/main/resources/org/jenkinsci/plugins/configfiles/ConfigFilesUI/setup-codemirror.js

```diff
@@ -0,0 +1,14 @@
+document.addEventListener("DOMContentLoaded", function() {
+    const textarea = document.querySelector("textarea#config\\.content");
```

Comment:
Seems overly complex. The original code from the Jelly files is simple and concise, as well as avoiding the need for complex escaping:

```suggestion
    const textarea = document.getElementById("config.content");
```